src/main/js/field/input/TableSelectInput.js

```diff
@@ -10,32 +10,44 @@ import { createNewConfigurationRequest } from 'util/configurationRequestBuilder'
 import PropTypes from 'prop-types';
 import { Modal } from "react-bootstrap";
 
-const { Option, SingleValue, ValueContainer } = components;
+const { MultiValue, ValueContainer } = components;
 
-const typeOptionLabel = props => (
-    <Option {...props}>
-        <DescriptorOption icon={props.data.icon} label={props.data.label} value={props.data.value} />
-    </Option>
-);
+const typeLabel = (props) => {
+    const { data } = props;
+    const missingItem = (data.missing) ? { textDecoration: "line-through" } : {};
 
-const typeLabel = props => (
-    <SingleValue {...props}>
-        <DescriptorOption icon={props.data.icon} label={props.data.label} value={props.data.value} />
-    </SingleValue>
-);
+    return (
+        <MultiValue {...props}>
+            <DescriptorOption style={missingItem} icon={data.icon} label={data.label} value={data.value} />
+        </MultiValue>
+    );
+}
 
 const container = ({ children, getValue, ...props }) => {
     const length = getValue().length;
-    return (length <= 5) ?
-        <ValueContainer {...props}>
-            {children}
-        </ValueContainer>
-        :
+    const error = (
+        <span className="missingBlackDuckData">
```

Comment:
We should rename the CSS class missingBlackDuckData. This field can apply to more than just black duck data